precode.sh

```diff
@@ -6,38 +6,55 @@
 #   dir2
 #   dir3
 #       dir4
+mkdir -p 'task/dir1 task/dir2 task/dir3/dir4'
```

Comment:
С кавычками не будет работать
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```